cellxgene_schema_cli/scripts/gtf_tools.py

```diff
@@ -0,0 +1,76 @@
+from typing import List, Tuple
+
+
+def merge_bed_ranges(ranges: List[Tuple]) -> List[Tuple]:
+
+    """
+    Merges bed-like ranges and returns non-overlapping ranges
+
+    code adapted from http://www.genemine.org/gtftools.php
+
+    :param ranges List[Tuple]: List of ranges, each range is a tuple of the form [chromosome, start, end, strand]
+
+    :rtype List[Tuple]
+    :return A list of non-overlapping ranges
+    """
+
+    ranges.sort(key=lambda x: (x[0], x[1]))
+    merged = []
+    n_range = len(ranges)
+    if n_range == 1:
+        merged = ranges
+    elif n_range == 2:
+        imerge = _neighbor_merge_ranges(ranges[0], ranges[1])
+        for each in imerge:
+            merged.append(each)
+    else:
+        i = 2
+        imerge = _neighbor_merge_ranges(ranges[0], ranges[1])
+        n_imerge = len(imerge)
+        while n_imerge > 0:
+            if n_imerge == 2:
+                merged.append(imerge[0])
+
+            imerge = _neighbor_merge_ranges(imerge[n_imerge - 1], ranges[i])
+            n_imerge = len(imerge)
+            if i == n_range - 1:
+                for each in imerge:
+                    merged.append(each)
+                n_imerge = -1
+            i += 1
+
+    return merged
+
+
+def _neighbor_merge_ranges(range1, range2):
+
+    """
+    Merges two neighboring ranges
```

Comment:
Can you include params/return values?
Something like
```
    : param range1 List[Tuple]: List of ranges, each range is a tuple of the form [chromosome, start, end, strand]
    : param range2 List[Tuple]: List of ranges, each range is a tuple of the form [chromosome, start, end, strand]
    
    :rtype List[Tuple]
    :return A list of non-overlapping ranges
```
^ based off of merge_bed_ranges